_episodes/020-mun.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Mun" 
+date: 2020-06-26T17:00:00+00:00
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e016-mun.mp3
+duration: "00:50:38"
+length: "40029157"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+First time host, long time editor Jeremy talks with Bas & Remco, creators of the Mun project. Mun is a programming language empowering creation through speedy, hot reloading iteration written in Rust. Why Rust for a project like this? that's what we explore.
```

Comment:
It looks like iTunes doesn't like the `&amp;` that gets generated for the & here. Could you replace it with "and"?

Oh, and capitalize the "T" in "that's what we explore"? And maybe add "in this episode" to the end of that?